samples/widgets/editor/src/runtime/widget.tsx

```diff
@@ -109,9 +109,13 @@ export default class Widget extends React.PureComponent<AllWidgetProps<{}>, Stat
     return (
       <div
         className="widget-js-api-editor"
-      css={[`height: "100%"; overflow: "auto";`, css]}
-    >
-     <div ref={this.myRef}></div>
+        style={{ height: "100%" , overflow: "auto" }}
+      >
+       <div className="here" ref={this.myRef}>
```

Comment:
```suggestion
       <div ref={this.myRef}>
```

we don't need this class, right?

Reply:
right we don't need it that was left over from testing.